full-service/src/service/ledger.rs

```diff
@@ -139,7 +144,7 @@ where
                 .filter_map(|conn| conn.fetch_block_info(empty()).ok())
                 .map(|block_info| block_info.minimum_fee)
```

Comment:
Eran pointed out the fix is incomplete, because protobuf will default the minimum_fee member to zero, not `None`:

```suggestion
                .filter_map(|block_info| {
                    // Cleanup the protobuf default fee
                    if block_info.minimum_fee == 0 {
                        None
                    } else {
                        Some(block_info.minimum_fee)
                    }
                })
```